src/Microsoft.Health.Blob/Features/Storage/BlobClientReadWriteTestProvider.cs

```diff
@@ -37,12 +46,20 @@ public async Task PerformTestAsync(BlobServiceClient client, BlobContainerConfig
             BlobContainerClient blobContainer = client.GetBlobContainerClient(blobContainerConfiguration.ContainerName);
             BlockBlobClient blob = blobContainer.GetBlockBlobClient(TestBlobName);
 
-            using var content = new MemoryStream(Encoding.UTF8.GetBytes(TestBlobContent));
-            await blob.UploadAsync(
-                content,
-                new BlobHttpHeaders { ContentType = "text/plain" },
-                cancellationToken: cancellationToken).ConfigureAwait(false);
-            await DownloadBlobContentAsync(blob, cancellationToken).ConfigureAwait(false);
+            try
+            {
+                _logger.LogInformation("Reading and writing blob: {container}/{blob}", blobContainerConfiguration.ContainerName, TestBlobName);
+                using var content = new MemoryStream(Encoding.UTF8.GetBytes(TestBlobContent));
+                await blob.UploadAsync(
+                    content,
+                    new BlobHttpHeaders { ContentType = "text/plain" },
+                    cancellationToken: cancellationToken).ConfigureAwait(false);
+                await DownloadBlobContentAsync(blob, cancellationToken).ConfigureAwait(false);
+            }
+            catch (OperationCanceledException ex)
+            {
+                _logger.LogCritical(ex, "Reading and writing cancelled for blob: {container}/{blob}", blobContainerConfiguration.ContainerName, TestBlobName);
```

Comment:
Should we instead move this logging to the top of the call stack? Unfortunately I don't know where this is called. I am unsure if we invoke this method. Perhaps I am just blind lol

Reply:
fixed